test/TestHelpers.jl

```diff
@@ -216,6 +216,8 @@ end
     @inbounds deleteat!(parent(A), first_idx:last_idx)
 end
 
+Base.resize!(A::OffsetVector, nl::Integer) = resize!(A.parent, nl)
```

Comment:
Though `resize!` should return `A` rather than the parent, so it should be defined as `resize!(...) = (resize!(A.parent, nl); A)`.